### PR TITLE
Added docstrings to aireos_device.py

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

File: pyntc/devices/aireos_device.py [hunk 1]
```diff
@@ -467,7 +486,7 @@ def close(self):
             self._connected = False
 
     def config(self, command, **netmiko_args):
-        """
+        r"""
```
Comment (mundruid) [inline pyntc/devices/aireos_device.py:489]: Seems like this passes all tests but there is an extra `r"""` in front of the docstring.

**Review by mundruid — APPROVED** [review]

File: pyntc/devices/aireos_device.py (hunk 1 above)
Comment (qduk) [reply pyntc/devices/aireos_device.py:489]: Hey Dr.X, pydocstyle recommends putting an r before docstrings that contains backslashes. It is error D301 on this page.

**Review by mundruid — APPROVED** [review]
Thank you for all the work adding docstrings @qduk

File: pyntc/devices/aireos_device.py [hunk 2]
```diff
@@ -600,9 +619,9 @@ def confirm_is_active(self):
         return True
 
     @property
-    def connected(self):
+    def connected(self):  # noqa: D401
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:622]:
```suggestion
    def connected(self):
```

File: pyntc/devices/aireos_device.py [hunk 3]
```diff
@@ -600,9 +619,9 @@ def confirm_is_active(self):
...
         """
-        The connection status of the device.
+        Connection status of the device.
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:624]:
```suggestion
        Get connection status of the device.
```

File: pyntc/devices/aireos_device.py [hunk 4]
```diff
@@ -657,9 +676,9 @@ def disable_wlans(self, wlan_ids):
                 raise WLANDisableError(self.host, desired_wlans, post_disabled_wlans)
 
     @property
-    def disabled_wlans(self):
+    def disabled_wlans(self):  # noqa:  D403
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:679]:
```suggestion
    def disabled_wlans(self):  # noqa: D403
```

File: pyntc/devices/aireos_device.py [hunk 5]
```diff
@@ -741,9 +760,9 @@ def enable_wlans(self, wlan_ids):
                 raise WLANEnableError(self.host, desired_wlans, post_enabled_wlans)
 
     @property
-    def enabled_wlans(self):
+    def enabled_wlans(self):  # noqa:   D403
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:763]:
```suggestion
    def enabled_wlans(self):  # noqa: D403
```

File: pyntc/devices/aireos_device.py [hunk 6]
```diff
@@ -1056,7 +1092,7 @@ def handler(signum, frame):
     @property
     def redundancy_mode(self):
         """
-        The oprating redundancy mode of the controller.
+        Operating redundancy mode of the controller.
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:1095]:
```suggestion
        Get operating redundancy mode of the controller.
```

File: pyntc/devices/aireos_device.py [hunk 7]
```diff
@@ -1222,9 +1270,10 @@ def show(self, command, expect_string=None, **netmiko_args):
 
         return command_responses
 
-    def show_list(self, commands, **netmiko_args):
+    def show_list(self, commands, **netmiko_args):  # noqa:  D401
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:1273]:
```suggestion
    def show_list(self, commands, **netmiko_args):  # noqa: D401
```

File: pyntc/devices/aireos_device.py [hunk 8]
```diff
@@ -1255,6 +1304,12 @@ def show_list(self, commands, **netmiko_args):
 
     @property
     def startup_config(self):
+        """
+        Show startup config.
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:1308]:
```suggestion
        Get startup config.
```

File: pyntc/devices/aireos_device.py [hunk 9]
```diff
@@ -1334,7 +1389,7 @@ def transfer_image_to_ap(self, image, timeout=None):
     @property
     def uptime(self):
         """
-        The uptime of the device in seconds.
+        Uptime of the device in seconds.
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:1392]:
```suggestion
        Get uptime of the device in seconds.
```

File: pyntc/devices/aireos_device.py [hunk 10]
```diff
@@ -1354,8 +1409,7 @@ def uptime(self):
     @property
     def uptime_string(self):
         """
-        The uptime of the device as a string.
-        The format is dd::hh::mm
+        Uptime of the device as a string in the format is dd::hh::mm.
```
Comment (jmcgill298) [inline pyntc/devices/aireos_device.py:1412]:
```suggestion
        Get uptime of the device as a string in the format is dd::hh::mm.
```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**